core/src/test/java/org/infinispan/protostream/MarshallingTest.java

```diff
@@ -62,6 +62,7 @@ public void testMarshallAccount() throws Exception {
       Account account = new Account();
       account.setId(1);
       account.setDescription("test account");
+      account.setCurrencies(new Account.Currency[]{Account.Currency.BRL});
       Date creationDate = new Date();
```

Comment:
Could you add an assert that the enum value correclty survives marshaling and unmarshalling?